src/rubicon/objc/api.py

```diff
@@ -246,28 +246,34 @@ def __repr__(self):
         return f"{type(self).__qualname__}({self.name_start!r})"
 
     def __call__(self, receiver, first_arg=_sentinel, **kwargs):
-        if first_arg is ObjCPartialMethod._sentinel:
+        # Ignore parts of argument names after "__".
+        order = [argname.split("__")[0] for argname in kwargs]
```

Comment:
As mentioned below - given that `order` is only ever used as a tuple, it makes sense to construct it as one.
```suggestion
        order = tuple(argname.split("__")[0] for argname in kwargs)
```